dali/imgcodec/decoders/nvjpeg2k/nvjpeg2k.h

```diff
@@ -86,7 +86,7 @@ class DLL_PUBLIC NvJpeg2000DecoderInstance : public BatchParallelDecoderImpl {
     }
   };
 
-  static constexpr int kNumParallelTiles = 1;   // TODO(michalz): Use a different memory resource
+  static constexpr int kNumParallelTiles = 2;   // TODO(michalz): Use a different memory resource
```

Comment:
2 is enough?

Reply:
I think more is too much - each tile has its own state and associated memory consumption.